.github/CODEOWNERS

```diff
@@ -7,8 +7,17 @@
 # review when someone opens a pull request.
 *       @aws/dl-containers
 
+
+# Common files that is managed by multiple owners
+available_images.md @aws/dl-containers @aws/dlc-autogluon-reviewers @aws/dlc-neuron-reviewers @aws/dlc-eia-reviewers @aws/dlc-dgl-reviewers @aws/dlc-trcomp-reviewers
```

Comment:
Can you remove dgl (deep graph library) reviewers from this file? I do not think it is needed anymore

Reply:
done